docs/usage.rst

```diff
@@ -87,6 +87,30 @@ datasets, you can use **--output-value** command-line option:
     $ cernopendata-client get-metadata --recid 1 --output-value system_details.global_tag
     FT_R_42_V10A::All
 
+If you would like to extract parts of the metadata by filtering some
+field, for example to extract all of the affiliations of the authors, 
+or all the data related to a certain author, or just the orcid of the 
+author, you can use the **--filter** command-line option:
+
+.. code-block:: console
+
+    $ cernopendata-client get-metadata --recid 329 --output-value authors.orcid
+    0000-0002-2634-4958
+    0000-0001-8363-9827
+    0000-0002-9764-9783
+    0000-0002-9266-1783
+    0000-0001-5610-1060
+    0000-0001-7613-8063
+    $ cernopendata-client get-metadata --recid 329 --output-value authors --filter name='Rousseau, David'    
```

Comment:
This one comes "in the middle" of the ORCID example. I think we have two options:

1. First show example getting all authors, and filtering on one. Then show another example how to get only ORCID identifiers.
2. Alternatively, we don't show full author JSON output, only the ORCID filtering.